syntax = "proto3";
package role;





/*
RoleAddUser请求
*/
message RoleAddUserRequest {
    /*
    用户名列表
    */
    repeated string operate_user = 1 ;
    /*
    角色id
    */
    string id = 2 ;
}

/*
RoleAddUser返回
*/
message RoleAddUserResponse {
    /*
    新加用户数量
    */
    int32 count = 1 ;
}

/*
RoleAddUserApi返回
*/
message RoleAddUserResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    role.RoleAddUserResponse data = 4 ;
}

